src/classes/STG_InstallScript.cls

```diff
@@ -228,6 +233,33 @@ global without sharing class STG_InstallScript implements InstallHandler {
         upsert orgSettings;
     }
 
+    /*******************************************************************************************************
+    * @description Enabled Auto Population of Affiliation on Student Course Connection.
+    * @return void
+    */
+    global static void enableAutoPopulationOfAffilOnStudentCourseCxn() {
+        Hierarchy_Settings__c orgSettings = UTIL_CustomSettingsFacade.getSettings();
+        if (orgSettings.Populate_Affil_on_Student_Couse_Cxn__c == null || orgSettings.Populate_Affil_on_Student_Couse_Cxn__c != true) {
```

Comment:
did we spell Course Connection (Couse_Cxn) wrong when we added it to Hierachy Settings? bummer :)

Reply:
@lifewithryan , yeah :( But the label is correct. ;/